apps/www/app/glossary/client.tsx

```
@@ -0,0 +1,232 @@
+                    {letter}
+                  </span>
+                ),
+              )}
+            </div>
+            {Object.entries(groupedTerms).map(
+              ([letter, letterTerms]) =>
+                letterTerms.length > 0 && (
+                  <section key={letter} id={letter} className="mb-8 scroll-mt-32">
+                    <h2 className="text-2xl font-semibold mb-4 grid-cols-1">{letter}</h2>
+                    <div className="grid grid-cols-1 gap-8 auto-rows-fr xl:grid-cols-3 md:grid-cols-2 grid-col-1">
+                      {letterTerms.map(({ slug, image, category, title, description }) => (
```

Biome

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)